src/test/java/com/epam/test_generator/services/SuitServiceTest.java

```diff
@@ -139,4 +137,96 @@ public void removeSuitTest_expectNotFoundException() {
         suitService.removeSuit(SIMPLE_SUIT_ID);
     }
 
+    @Test
+    public void updateSuitRowNumber_SimpleSuit_StatusOk() {
+        final SuitRowNumberUpdateDTO update = getRowNumberUpdateDTO(5L, 2);
+        final SuitRowNumberUpdateDTO update1 = getRowNumberUpdateDTO(4L, 1);
+
+        List<SuitRowNumberUpdateDTO> rowNumbers = new ArrayList<>(
+                Arrays.asList(update, update1)
+        );
+
+        List<Suit> suits = new ArrayList<>(Arrays.asList(
+                new Suit(
+                        4L,
+                        "name",
+                        "description",
+                        new ArrayList<>(),
+                        3,
+                        new HashSet<>(),
+                        2
+                ),
+                new Suit(
+                        5L,
+                        "name2",
+                        "description2",
+                        new ArrayList<>(),
+                        2,
+                        new HashSet<>(),
+                        1
+                )
+        ));
+
+        final Map<Long, Integer> patch = rowNumbers
+                .stream()
+                .collect(Collectors.toMap(SuitRowNumberUpdateDTO::getId, SuitRowNumberUpdateDTO::getRowNumber));
+
+        when(suitDAO.findByIdInOrderById(patch.keySet())).thenReturn(suits);
+        suitService.updateSuitRowNumber(rowNumbers);
+        assertEquals(1, suits.get(0).getRowNumber());
+        assertEquals(2, suits.get(1).getRowNumber());
+    }
+
+    private SuitRowNumberUpdateDTO getRowNumberUpdateDTO(Long suitId, Integer rowNumber) {
+        final SuitRowNumberUpdateDTO update = new SuitRowNumberUpdateDTO();
+        update.setId(suitId);
+        update.setRowNumber(rowNumber);
+        return update;
+    }
+
+
+    @Test(expected = BadRequestException.class)
+    public void updateSuitRowNumber_throwsException_ThereIsNoSuchSuitWithSuchId() {
+
+        final SuitRowNumberUpdateDTO update = getRowNumberUpdateDTO(5L, 2);
+
+        final SuitRowNumberUpdateDTO update1 = getRowNumberUpdateDTO(4L, 1);
+
+        final List<SuitRowNumberUpdateDTO> rowNumbers = new ArrayList<>(
```

Comment:
I can't spot any reason to use Arrays.asList as source to ArrayList.

Reply:
Done